src/core/public/chrome/ui/header/nav_link.test.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { isActiveNavLink, createEuiListItem } from './nav_link';
+import { isModifiedOrPrevented } from './nav_link';
+import { ChromeNavLink } from '../../..';
+import { httpServiceMock } from '../../../http/http_service.mock';
+import React from 'react';
+
+describe('isModifiedOrPrevented', () => {
+  test('returns true if metaKey is pressed', () => {
+    const mockEvent: Partial<React.MouseEvent<HTMLButtonElement, MouseEvent>> = {
+      metaKey: true,
```

Comment:
NIT: I don't think we need to test `isModifiedOrPrevented` function in ut since it is a straightforward boolean expression with no complex logic.